### PR TITLE
LPS-126300

### PR DESCRIPTION
**Pull request by alexcandidos**

### CODE REVIEW

**Comment by Tim-Cao** [comment]
Hi @alexcandidos, this case is already covered by LocalizedSimpleFieldsetsStructure#TranslateFieldsetWithNestedFields

**Comment by emanuelfisilva** [comment]
Thanks for your review @Tim-Cao , I agree with the duplication, but I'll choose to keep this test and discard the other.

File: portal-web/test/functional/com/liferay/portalweb/macros/DataEngine.macro [hunk 1]
```diff
@@ -1157,6 +1157,18 @@ definition {
 			locator1 = "Sidebar#FIELDSET_ITEM_ELLIPSIS");
 	}
 
+	macro gotoNewFieldset {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/DataEngine.macro:1160]: Use existing macro

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/dynamicdatamapping/DDMEditStructure.path [hunk 2]
```diff
@@ -208,6 +208,11 @@
 	<td>//label[normalize-space(text())='${key_fieldsGroupLabel}']/parent::div/parent::div[contains(@class,'form-group')]</td>
 	<td></td>
 </tr>
+<tr>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/dynamicdatamapping/DDMEditStructure.path:211]: Use existing locator

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3783
View total diff https://github.com/liferay-echo/liferay-portal/pull/3783/commits/c10e480a04595eb1e5d13c413b88ea773effe30c

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**